src/Nethermind/Nethermind.JsonRpc/Data/BlockParameterConverter.cs

```diff
@@ -1,150 +0,0 @@
-// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
-// SPDX-License-Identifier: LGPL-3.0-only
-
-using System;
-using Nethermind.Blockchain;
-using Nethermind.Blockchain.Find;
-using Nethermind.Core.Crypto;
-using Nethermind.JsonRpc.Modules.Trace;
-using Nethermind.Serialization.Json;
-using Newtonsoft.Json;
-
-namespace Nethermind.JsonRpc.Data
```

Comment:
Benefit of this approach was that Json serialization/deserialization was decoupled from domain types.